pkg/cmd/render/render.go

```diff
@@ -52,6 +63,16 @@ func (r *renderOpts) AddFlags(fs *pflag.FlagSet) {
 	r.generic.AddFlags(fs, configv1.GroupVersion.WithKind("Config"))
 
 	fs.StringVar(&r.clusterConfigFile, "cluster-config-file", r.clusterConfigFile, "Openshift Cluster API Config file.")
+
+	// This is the file containing the infrastructure object
+	fs.StringVar(&r.clusterInfrastructureFile, "cluster-infrastructure-file", r.clusterInfrastructureFile, "Input path for the cluster infrastructure file.")
```

Comment:
input or output file?  Either we should have that postfix everywhere or nowhere.

Reply:
I've changed the names to reflect input/output.